packages/down-up-mixin/test/down-up-mixin.test.ts

```diff
@@ -0,0 +1,184 @@
+import { LitElement, html, customElement } from 'lit-element';
+import { fixture } from '@open-wc/testing-helpers';
+import { mousedown, mouseup, touchstart, touchend } from '@vaadin/test-helpers';
+import { DownUpMixin } from '../down-up-mixin';
+
+const { expect } = chai;
+const { sinon } = window;
+
+@customElement('dum-element')
+class DumElement extends DownUpMixin(LitElement) {
+  render() {
+    return html`
+      <div>Content</div>
+    `;
+  }
+
+  protected _onDown() {
+    this.dispatchEvent(new CustomEvent('down'));
+  }
+
+  protected _onUp() {
+    this.dispatchEvent(new CustomEvent('up'));
+  }
+}
+
+@customElement('dum-child-element')
+class DumChildElement extends DownUpMixin(LitElement) {
+  render() {
+    return html`
+      <div
+        @mousedown="${this._onMouseDown}"
+        @touchstart="${this._onTouchStart}"
+        @touchend="${this._onTouchEnd}"
+      >
+        Content
+      </div>
+    `;
+  }
+
+  protected get _downUpTarget() {
+    return null;
+  }
```

Comment:
It doesn't feel right to me that you need to explicitly set all these event listeners when the intention is just to override `_downUpTarget` to some element in the component template. There should be a simpler way to do this, preferably by simply overriding `_downUpTarget` somewhere.

Also making `_downUpTarget` to return null could cause problems in the future if the mixin logic is somehow updated to use it in different ways.

It would be great if this test could be a good (best practice) example of how to do this kind of override of `_downUpTarget`.

I'm not sure what is the best way to do this however since it needs to be overridden after the template has been rendered but before the mixin tries to add the event listeners.

This could be done by providing a new hook in DownUp mixin (see my suggestion there too).
I tested that the tests still pass with these changes here and in DownUp mixin.

```suggestion
  render() {
    return html`
      <div>
        Content
      </div>
    `;
  }

  protected _initDownUpTarget() {
    this._downUpTarget = this.renderRoot.querySelector('div');
  }
```

Reply:
@Haprog I would prefer to keep using a getter, to be consistent with `focusElement` which we have in `ControlStateMixin`. But I agree that tests can be adjusted as you suggest.